package.json

```diff
@@ -20,10 +20,14 @@
   "repository": "nanostores/preact",
   "sideEffects": false,
   "type": "module",
-  "types": "./index.d.ts",
+  "types": "./src/index.d.ts",
   "exports": {
-    ".": "./index.js",
-    "./package.json": "./package.json"
+    ".": "./src/index.js",
+    "./package.json": "./package.json",
+    "./signals": {
```

Comment:
Why do we need extra export?

Reply:
I used tree shaking to isolate the storage as a signal in case the developer uses preact without signals.